package.json

```diff
@@ -29,9 +29,9 @@
     "@storybook/addon-links": "6.5.0-alpha.16",
     "@storybook/react": "6.5.0-alpha.16",
     "@storybook/testing-react": "1.2.3",
-    "@testing-library/jest-dom": "5.16.1",
-    "@testing-library/react": "12.1.2",
-    "@testing-library/user-event": "13.5.0",
+    "@testing-library/jest-dom": "^5.16.4",
```

Comment:
Update @testing-library to latest version to fix the following error when running UT
```
ReactDOM.render is no longer supported in React 18. Use createRoot instead.
```